ext/opentelemetry-ext-http-requests/src/opentelemetry/ext/http_requests/__init__.py

```diff
@@ -72,6 +74,11 @@ def instrumented_request(self, method, url, *args, **kwargs):
             # TODO: Propagate the trace context via headers once we have a way
             # to access propagators.
 
+            headers = kwargs.get("headers", {})
```

Comment:
Instead of get + set, use [`kwargs.setdefault`](https://docs.python.org/3/library/stdtypes.html#dict.setdefault).